src/components/layers/overlays/OverlayCard.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import i18n from '@dhis2/d2-i18n';
+import LayerCard from '../LayerCard';
+import Legend from '../../legend/Legend';
+import {
+    editLayer,
+    removeLayer,
+    changeLayerOpacity,
+    toggleLayerExpand,
+    toggleLayerVisibility,
+} from '../../../actions/layers';
+import { setMessage } from '../../../actions/message';
+import { toggleDataTable } from '../../../actions/dataTable';
+import { openDataDownloadDialog } from '../../../actions/dataDownload';
+import { setAnalyticalObjectAndSwitchApp } from '../../../util/analyticalObject';
+import styles from './styles/OverlayCard.module.css';
+
+const downloadableLayerTypes = ['facility', 'thematic', 'boundary', 'event'];
```

Comment:
What about having a layerTypes constants file that exports all of these?

Reply:
layer types constants are now added to contstants/layers.js